packages/apps/transfer/src/pages/transfer/account-transactions/index.tsx

```diff
@@ -131,7 +131,7 @@ const CheckTransactions: FC = () => {
             <StyledSmallField>
               <Select
                 leadingText={t('Chain')}
-                onChange={(e) => setChain(e.target.value)}
+                onChange={(e: any) => setChain(e.target.value)}
```

Comment:
how about using
`onChange={(e: SyntheticEvent<HTMLSelectElement>) =>`

Reply:
yeah that was already changed in other PR. I just wanted my build to succeed for a test :)
Thanks it should be good now